app/tools/jwtToken.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken'),
```

Comment:
we dont use tools, we use helpers, but its ok for now

Reply:
👍